### PR TITLE
Fixing a couple of things in the example, and bumping docker version

### LINKED ISSUE #40435: Update cassandra container to newer ubuntu version
The container used as the base image https://github.com/kubernetes/kubernetes/blob/master/examples/storage/cassandra/image/Dockerfile

Has to be updated.  The pgp command to import the keys is not working as well.  We a general update of the container Dockerfile and build.

Currently we are unable to build a new C* container.

### PR DESCRIPTION
**Pull request by chrislovecnm**
**What this PR does / why we need it**:

Updates the Cassandra version, and fixes a couple of issues in the example documentation.

fixes #40435

### CODE REVIEW

**Comment by chrislovecnm** [comment]
@zmerlynn can you push the container please

File: examples/storage/cassandra/image/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright 2016 The Kubernetes Authors.
+# Copyright 2017 The Kubernetes Authors.
```
Comment (zmerlynn) [inline examples/storage/cassandra/image/Dockerfile]: Nit: Once committed, don't muck with the copyright.

**Comment by zmerlynn** [comment]
/lgtm
/approve

I'm working on figuring out who actually has rights to `google-samples`.

**Comment by zmerlynn** [comment]
/undo lgtm
Oh, fix the nit before we commit it.

**Comment by zmerlynn** [comment]
Undoing /lgtm until nit is fixed. Feel free to self-apply after it's fixed.

**Comment by zmerlynn** [comment]
cc @jeffmendoza 

I apparently don't have rights in `google-samples`. I'd be happy to figure out how to fix that, but in the meantime, CC'ing one of the people I know has rights.

**Comment by jeffmendoza** [comment]
@zmerlynn @chrislovecnm `gcr.io/google_samples/cassandra:v12` is pushed from this branch. Did you need `-dev` pushed as well?

**Comment by zmerlynn** [comment]
@k8s-bot cvm gce e2e test this

**Comment by zmerlynn** [comment]
I'm adding this to the v1.6 milestone because it was almost in queue yesterday and just needed a sample pushed.

**Comment by chrislovecnm** [comment]
@k8s-bot cvm gce e2e test this

**Comment by zmerlynn** [comment]
@k8s-bot cvm gce e2e test this

**Comment by chrislovecnm** [comment]
This may not pass e2e until the container is bumped

**Pull request merged**